sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -176,6 +176,7 @@ class MapboxMap internal constructor(
     onStyleLoaded: Style.OnStyleLoaded? = null,
     onMapLoadErrorListener: OnMapLoadErrorListener? = null
   ) {
+    isStyleLoadInited = true
```

Comment:
Do we need to set it to false somewhere? in some destroy method.

Reply:
No, we will create new MapboxMap instance for new MapView